### DISCUSSION TITLE
What us tge beta get hub<title>

### DISCUSSION THREAD

**Discussion by jbird606** [community/community#69163] [Mobile]
### Select Topic Area

Product Feedback

### Body

I just saw where it said there was a beta for a new GitHub I clicked on it but it didn't really help me do anything wanted me to add a bug if I noticed one or to tell them how I liked it but I never got to see anything different but then again I really don't use the app that much I usually always use it on a laptop or desktop not so much on my phone

**Comment by stevepopovich** [comment]
Hi @jbird606, thanks for writing in. Do you have a specific bug or feature request to report?

**Reply by jbird606** [reply]
No sorry for wasting your time I just didn't understand at the time what it was saying
